pallets/funding/src/lib.rs

```diff
@@ -235,7 +235,7 @@ pub mod pallet {
 		type AuctionInitializePeriodDuration: Get<BlockNumberFor<Self>>;
 
 		/// The inner balance type we will use for all of our outer currency types. (e.g native, funding, CTs)
-		type Balance: Balance + From<u64> + FixedPointOperand + MaybeSerializeDeserialize + Into<u128>;
+		type Balance: Balance + From<u64> + FixedPointOperand + MaybeSerializeDeserialize + Into<u128> + Bounded;
```

Comment:
Where do we need the Bounded traitbound for?

Reply:
Good catch, it was needed in a previous version where we checked for close to overflow of Balance.